server/server.js

```diff
@@ -0,0 +1,57 @@
+import express from "express";
+import axios from "axios";
```

Comment:
I feel like I remember you mentioning that `axios` is a popular and well developed library compared to `fetch`, that does not have as many features associated with it and makes it a little more challenging to use. With `axios` you can use `async/await` to deal with asynchronous code instead of just promises. Also, `axios` can work in web browsers and Node.js. While `fetch` is supported in web browsers and Node, `axios` is still seen as more developer friendly. Some alternatives to `axios` (besides `fetch`) are jQuery Ajax and Superagent, I've never heard of these!